pkg/operator/ceph/cluster/osd/envs.go

```diff
@@ -196,3 +202,24 @@ func osdActivateEnvVar() []v1.EnvVar {
 
 	return append(cephVolumeEnvVar(), monEnvVars...)
 }
+
+func getTcmallocMaxTotalThreadCacheBytes(tcmallocMaxTotalThreadCacheBytes string) v1.EnvVar {
+	var value string
+	// If empty we read the default value from the file coming with the package
+	if tcmallocMaxTotalThreadCacheBytes == "" {
+		value = getTcmallocMaxTotalThreadCacheBytesFromFile()
```

Comment:
We read the file so we get whatever default value comes with the package. We have no idea what that default value should be and reading this file is easy enough and saves the burden of editing Rook each time it would change (although not often, but still).